Tests/Common.ps1

```diff
@@ -33,10 +33,15 @@ function Initialize-CommonTestSetup
     param()
 
     $moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
-    . (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Config\Settings.ps1')
+    $settingsPath = Join-Path -Path $moduleRootPath -ChildPath 'Tests\Config\Settings.ps1'
+    . $settingsPath
     Import-Module -Name (Join-Path -Path $moduleRootPath -ChildPath 'PowerShellForGitHub.psd1') -Force
 
-    if ([string]::IsNullOrEmpty($env:ciAccessToken))
+    $originalSettingsSha256Hash = "944D5BB450AD2C49F77DAE6C472FEF774B108CD2CB6A0DBF45EF4BBB7FE25D56"
+    $currentSettingsSha256Hash = (Get-FileHash -Path $settingsPath -Algorithm SHA256).Hash
+    $isSettingsUnaltered = $originalSettingsSha256Hash -eq $currentSettingsSha256Hash
+
+    if ([string]::IsNullOrEmpty($env:ciAccessToken) -and $isSettingsUnaltered)
```

Comment:
I think the more clear way to author this branching would be:

```
if ([string]::IsNullOrEmpty($env:ciAccessToken))
{
    if ($settingsIsUnaltered)
    {
    }
}
else
{
}
```

I think that reads better and it's easier to follow the branching logic that way.

Reply:
Haha ... Thanks! It's been updated.